yaksh/templates/exam.html

```diff
@@ -88,17 +89,39 @@
       <div class="panel-heading">Testcase No. {{ forloop.counter }}</div>
       <div class="panel-body">
       <div class="well well-sm">
-      {% if not error.expected_output %}
+      {% if not error.type %}
       <pre><code> {{error|safe}} </code></pre>
-      {% else %}
-        {% if error.given_input %}
-       <table class="table table-bordered">
-        <col width="30%">
-          <tr class = "active">
-          <td> For given Input value(s):</td>
-          <td>{{error.given_input}}</td>
+      {% elif error.type == 'assertion' %}
+            {% if error.test_case %}
+      <strong> We tried you code with the following test case:</strong><br/></br>
```

Comment:
Instead of `you` it should be `your`.  `We tried your code with the following test case:`

Reply:
resolved